libs/backend/src/cast_vote_records/export.ts

```diff
@@ -67,7 +67,9 @@ export interface ScannerStoreBase {
   clearCastVoteRecordHashes(): void;
   getBatches(): BatchInfo[];
   getCastVoteRecordRootHash(): string;
-  getElectionDefinition(): ElectionDefinition | undefined;
+  getElectionRecord():
+    | { electionDefinition: ElectionDefinition; electionPackageHash: string }
```

Comment:
why not have some kind of ElectionDefinitionWithHash interface for this type you are using around?

Reply:
Yeah, I considered doing that but wasn't sure how often I would need it. I think in retrospect it's worthwhile, I'll add it